pallets/committee/src/lib.rs

```diff
@@ -263,16 +257,15 @@ decl_module! {
 
         fn deposit_event() = default;
 
-        /// Change the vote threshold the determines the winning proposal. For e.g., for a simple
-        /// majority use (1, 2) which represents the in-equation ">= 1/2"
+        /// Change the vote threshold the determines the winning proposal.
+        /// For e.g., for a simple majority use (1, 2) which represents the in-equation ">= 1/2".
         ///
         /// # Arguments
-        /// * `match_criteria` - One of {AtLeast, MoreThan}.
         /// * `n` - Numerator of the fraction representing vote threshold.
         /// * `d` - Denominator of the fraction representing vote threshold.
         #[weight = <T as Trait<I>>::WeightInfo::set_vote_threshold()]
         pub fn set_vote_threshold(origin, n: u32, d: u32) {
-            T::CommitteeOrigin::ensure_origin(origin)?;
+            T::VoteThresholdOrigin::ensure_origin(origin)?;
```

Comment:
Its not uniform yes, but the goal here is to enable a different origin than the committee VMO to change the vote threshold since that is somewhat sensitive. For example, we might only want to allow the GC VMO to set the vote threshold for the technical committee. In practice, it currently holds that `type VoteThresholdOrigin = Self::CommitteeOrigin;` so this has no semantic implications for the time being, only opens up for that later.
We could also add more associated types for those other extrinsics, but I don't see the need right now.